pkg/pipelines/eventlisteners/eventlisteners_test.go

```diff
@@ -63,10 +61,18 @@ func TestGenerateEventListener(t *testing.T) {
 								},
 							},
 						},
+						{
+							CEL: &triggersv1.CELInterceptor{
+								Filter: "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == 'org/test') && body.ref.endsWith(body.repository.default_branch)",
```

Comment:
`body.ref.endsWith(body.repository.default_branch)` feels a bit loose, maybe `body.ref == 'refs/heads/' + body.repository.default_branch` ?

Reply:
Updated for both GitHub and GitLab